pkg/scrape/vrbangers.go

```diff
@@ -28,6 +28,9 @@ func VRBangersSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, ou
 		sc.Site = siteID
 		sc.HomepageURL = strings.Split(e.Request.URL.String(), "?")[0]
 
+		if e.ChildAttr(`link[rel=shortlink]`, "href") == "" {
+			return
```

Comment:
Looks like this was a temporary glitch as the VRBangers scraper no longer crashes.

Maybe add some logging so that we can at least see that the scene is skipped?

```suggestion
			log.Printf("Skipping %s because it is not a valid scene.", e.Request.URL.String())
			return
```